tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,974 @@
+{
+  "imagemagick": [
+    {
+      "reason_to_ignore": "No fix at this time, Package is at highest version for OS",
+      "description": " In ImageMagick, there is load of misaligned address for type 'double', which requires 8 byte alignment and for type 'float', which requires 4 byte alignment at MagickCore/property.c. Whenever crafted or untrusted input is processed by ImageMagick, this causes a negative impact to application availability or other problems related to undefined behavior.",
+      "vulnerability_id": "CVE-2022-32547",
+      "name": "CVE-2022-32547",
+      "package_name": "imagemagick",
+      "package_details": {
+        "file_path": null,
+        "name": "imagemagick",
+        "package_manager": "OS",
+        "version": "6.9.10.23+dfsg",
+        "release": "2.1ubuntu11.4"
+      },
+      "remediation": { "recommendation": { "text": "None Provided" } },
+      "cvss_v3_score": 7.8,
+      "cvss_v30_score": 0.0,
+      "cvss_v31_score": 7.8,
+      "cvss_v2_score": 6.8,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-32547.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-32547 - imagemagick-6-common, libmagickcore-6.q16-6 and 3 more"
+    },
+    {
+      "description": " In ImageMagick, there is load of misaligned address for type 'double', which requires 8 byte alignment and for type 'float', which requires 4 byte alignment at MagickCore/property.c. Whenever crafted or untrusted input is processed by ImageMagick, this causes a negative impact to application availability or other problems related to undefined behavior.",
```

Comment:
The json record here and the one above seem to be the same and missing reason_to_ignore. Any reason for the record duplication and omission of the key "reason_to_ignore"?

Reply:
Only doing for the package and not every as we cannot put next to the package due to the json structure, this is only provided to the top most CVE so it is clearly seen for the package. Any more and we are just making unneeded noise and work as it's the package that needs to be updated.